### ISSUE TITLE
Replacing colon with dash

### ISSUE THREAD

**Issue by TheOttoman2012** [mylar3/mylar3#578]
This is basically open issue #987 from the previous version of mylar (https://github.com/evilhero/mylar/issues/987)

I would like to know if it is possible for Mylar to replace a colon with a dash in the title only when adding a new comic.

For example: If the comic title is: "Fantastic Four: House of M", Mylar currently handles the title as "Fantastic Four House of M". It should be "Fantastic Four - House of M"

**Comment by evilhero** [comment]
This is included as part of the preview renamer option that should be coming soon (hopefully before the end of year). 

I have to figure out some nagging display issues when viewing filenames that span more than one page, but aside from that it's ready to go - pending testing, and free time of course.

**Comment by leaderdog** [comment]
I'm just curious as to how this is coming along?  Also, once it's implemented, will I be able to hit rename in manage comics and all the books will add the missing dash that was removed?

**Comment by evilhero** [comment]
@leaderdog, yes basically. However there are few bigger items I have that I need to get out prior to this one - unless something else happens, this is still a little bit aways (not like years, like maybe a few weeks at most maybe?)

**Comment by mouzzampk2014** [comment]
Thank you so much. I am also keeping an eye on this feature like a hawk :) thanks

**Comment by ajhorvath20** [comment]
Hello, I'm just wondering if there amis any update on this? 

Thank you so much

**Comment by barbequesauce** [comment]
Closing due to staleness, we welcome contributions for new features. If this is an issue and you are still experiencing it, please open a new ticket with a care package attached.
